include/mbgl/shaders/mtl/symbol.hpp

```diff
@@ -4,6 +4,8 @@
 #include <mbgl/shaders/shader_source.hpp>
 #include <mbgl/shaders/mtl/shader_program.hpp>
 
+#define MTL_SDF_SHADER_VERTEX_CULLING_ENABLED 1
```

Comment:
Note: before merging we should remove this `#define`.

@alexcristici Could you try this out and could give this PR a review?